explorer/src/lib/components/navbar/Navbar.svelte

```diff
@@ -114,8 +116,17 @@
             showSearchNotification = false;
           }}
           className="dusk-navbar__menu--search-notification"
+          {darkMode}
         />
       {/if}
+
+      <!-- No design implementation -->
+      <!-- <Switch
+        bind:value={isDarkMode}
+        on:change={() => {
+          appStore.setTheme(isDarkMode);
+        }}
+      /> -->
```

Comment:
Again, let's remove that, as we are not going to have a switch for the time-being.